### DISCUSSION TITLE
Floor plan Fixing nodes to image

### DISCUSSION THREAD

**Discussion by adamchwalisz** [visjs/vis-network#1877] [Q&A]
Hi,
I am looking for a way to achieve this 
https://gojs.net/latest/samples/kittenMonitor.html
with vis.js
Is it possible?

**Comment by nikhil-varma** [comment]
@adamchwalisz Assuming you have a way to show the background image, I feel that the points/nodes can basically be placed using the x and y attributes for the node itself? Maybe but could result in a lot of work. However my VisJs understanding is still rudimentary .
